source/includes/usage-examples/write-code-examples.java

```diff
@@ -0,0 +1,87 @@
+# start-insert-one
+Document document = new Document("<field name>", "<value>");
+Mono<InsertOneResult> result = Mono.from(collection.insertOne(document));
+
+result.doOnNext(i -> System.out.println(i.wasAcknowledged()))
+      .doOnError(e -> e.printStackTrace())
+      .doOnTerminate(mongoClient::close)
+      .subscribe();
+# end-insert-one
+
+# start-insert-multiple
+Document doc1 = new Document("<field name>", "<value>");
+Document doc2 = new Document("<field name>", "<value>");
+
+List<Document> documents = new ArrayList<Document>();
+
+documents.add(doc1);
+documents.add(doc2);
+
+Mono<InsertManyResult> result = Mono.from(collection.insertMany(documents));
+
+result.doOnNext(i -> System.out.println(i.wasAcknowledged()))
+      .doOnError(e -> e.printStackTrace())
+      .doOnTerminate(mongoClient::close)
+      .subscribe();
+# end-insert-multiple
+
+# start-update-one
+Mono<UpdateResult> result = Mono.from(collection.updateOne(eq("<field name>", "<value>"), set("<field name>", "<value>")));
```

Comment:
1. Let's avoid having lines that are so long that require a reader to scroll the code horizontally, as that makes examples much more difficult to read.
2. Also, if you are updating the value of `<field name>`, it makes sense to specify the new value different from the original one. For example, if the original value is `<value>`, then the new value can be `<new value>`. Let's not use the same keys and values when it does not make sense.

Both of the suggestions are applicable to many places in `write-code-examples.java`.

Something like the following will be better.

```java
Mono<UpdateResult> result = Mono.from(collection.updateOne(
        eq("<field name>", "<value>"),
        set("<field name>", "<new value>")));
```

Reply:
✅ Done.